### PR TITLE
NetApp

### PR DESCRIPTION
**Pull request by antleblanc**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | yes
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-5743
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description

NetApp GA

### :flags: Translations

- [x] TRANS-39046
- [x]  TRANS-39344

### CODE REVIEW

File: packages/manager/modules/netapp/src/dashboard/snapshot-policies/create/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  "netapp_snapshot_policies_create_months": "Mois",
+  "netapp_snapshot_policies_create_months_help": "Spécifie les mois d’exécution du programme de snapshots. Cette propriété est une liste de valeurs. La valeur minimale de cette propriété est de {{ min }} et la valeur maximale de {{ max }}.",
+  "netapp_snapshot_policies_create_weekdays": "Jour(s) de la semaine",
+  "netapp_snapshot_policies_create_weekdays_help": "Spécifie les jours de la semaine d’exécution du programme de snapshots. Cette propriété est une liste de valeurs. La valeur minimale de cette propriété est de {{ min }} et la valeur maximale de {{ max }}.",
+  "netapp_snapshot_policies_create_snapshots_to_keep": "Copies à garder",
+  "netapp_snapshot_policies_create_name": "Nom de la Snapshot Policy",
+  "netapp_snapshot_policies_create_description": "Description de la Snapshot policy",
+  "netapp_snapshot_policies_create_empty": "vide",
+  "netapp_snapshot_policies_create_rules": "Règles (UTC Time Zone): {{ count }}",
```
Comment (mohammed-zahaf) [inline packages/manager/modules/netapp/src/dashboard/snapshot-policies/create/translations/Messages_fr_FR.json:24]:
```suggestion
  "netapp_snapshot_policies_create_rules": "Règles (UTC Time Zone) : {{ count }}",
```

File: packages/manager/modules/netapp/src/dashboard/snapshot-policies/create/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  "netapp_snapshot_policies_create_add_rule": "Ajouter une nouvelle règle",
+  "netapp_snapshot_policies_create_submit": "Créer nouvelle Snapshot policy",
```
Comment (mohammed-zahaf) [inline packages/manager/modules/netapp/src/dashboard/snapshot-policies/create/translations/Messages_fr_FR.json:26]:
```suggestion
  "netapp_snapshot_policies_create_submit": "Créer une nouvelle Snapshot policy",
```

File: packages/manager/modules/netapp/src/dashboard/snapshot-policies/create/translations/Messages_fr_FR.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  "netapp_snapshot_policies_create_select_placeholder": "Choisissez…",
+  "netapp_snapshot_policies_create_selected_days": "{{ days }} jour(s)",
+  "netapp_snapshot_policies_create_selected_months": "{{ months }} mois",
+  "netapp_snapshot_policies_create_add_rule_validate": "Valider",
+  "netapp_snapshot_policies_create_add_rule_cancel": "Annuler",
+  "netapp_snapshot_policies_create_rule_delete": "Supprimer la règle",
+  "netapp_snapshot_policies_create_error": "Une erreur est survenue lors de la création de la Snapshot policy.",
```
Comment (mohammed-zahaf) [inline packages/manager/modules/netapp/src/dashboard/snapshot-policies/create/translations/Messages_fr_FR.json:33]: no API error message ?

File: packages/manager/modules/netapp/src/dashboard/snapshot-policies/translations/Messages_fr_FR.json [hunk 4]
```diff
@@ -1,3 +1,10 @@
 {
-  "netapp_snapshot_policies_breadcrumb": "Snapshot policies"
+  "netapp_snapshot_policies_breadcrumb": "Snapshot policies",
+  "netapp_snapshot_policies_title": "Gérer les Snapshot policies",
+  "netapp_snapshot_policies_information": "Créez et gérez facilement vos règles de snapshots, en matière de périodicité ou de durée de rétention. Vous pourrez ensuite les appliquer sur des volumes spécifiques. Les snapshots utilisent la capacité de stockage de votre solution Enterprise File Storage.",
+  "netapp_snapshot_policies_number": "Nombre de Snapshot policies",
+  "netapp_snapshot_policies_create": "Créer une nouvelle Snapshot policy",
+  "netapp_snapshot_policies_delete_success": "La Snapshot policy {{snapshotPolicyName}} a été supprimée avec succès !",
+  "netapp_snapshot_policies_delete_error": "Une erreur est survenue lors de la suppression de la Snapshot policy.",
+  "netapp_snapshot_policies_fetch_error": "Une erreur est survenue lors de la récupération de vos Snapshot policies."
```
Comment (mohammed-zahaf) [inline packages/manager/modules/netapp/src/dashboard/snapshot-policies/translations/Messages_fr_FR.json:9]: Same as here

File: packages/manager/modules/netapp/src/dashboard/volumes/dashboard/snapshots/Snapshot.class.js [hunk 5]
```diff
@@ -23,4 +23,8 @@ export default class Snapshot {
   isPending() {
     return STATUS.PENDING.includes(this.status);
   }
+
+  isBeingDeleted() {
```
Comment (mohammed-zahaf) [inline packages/manager/modules/netapp/src/dashboard/volumes/dashboard/snapshots/Snapshot.class.js:27]: may be we can rename it `isDeleting` to be compliant with otherers function (eg: `isPending`) ?

File: packages/manager/modules/netapp/src/dashboard/volumes/dashboard/snapshots/routing.js [hunk 6]
```diff
@@ -86,6 +87,26 @@ export default /* @ngInject */ ($stateProvider) => {
...
+
+      snapshotPolicies: /* @ngInject */ (getSnapshotPolicies) =>
+        getSnapshotPolicies().catch(() => []),
+      currentPolicy: /* @ngInject */ ($http, serviceName, volumeId) =>
+        $http
+          .get(
+            `/storage/netapp/${serviceName}/share/${volumeId}/snapshotPolicy`,
+          )
+          .then(({ data }) => data)
+          .catch(() => ({})),
```
Comment (mohammed-zahaf) [inline packages/manager/modules/netapp/src/dashboard/volumes/dashboard/snapshots/routing.js:99]: I think it's better to have HTTP request into a service file.

File: packages/manager/modules/netapp/src/dashboard/volumes/dashboard/snapshots/routing.js [hunk 7]
```diff
@@ -86,6 +87,26 @@ export default /* @ngInject */ ($stateProvider) => {
...
+
+      applyPolicy: /* @ngInject */ ($http, serviceName, volumeId) => (
+        snapshotPolicyID,
+      ) =>
+        $http.put(
+          `/storage/netapp/${serviceName}/share/${volumeId}/snapshotPolicy`,
+          {
+            snapshotPolicyID,
+          },
+        ),
```
Comment (mohammed-zahaf) [inline packages/manager/modules/netapp/src/dashboard/volumes/dashboard/snapshots/routing.js:109]: Same as previous

File: packages/manager/modules/netapp/src/order/controller.js [hunk 8]
```diff
@@ -130,6 +133,10 @@ export default class OvhManagerNetAppOrderCtrl {
   }
 
   goToOrderUrl() {
+    this.atInternet.trackClick({
+      name: `netapp::order::confirm::${this.selectedRegion}_${this.selectedLicense}_${this.selectedSize}TB_${this.duration}_${this.pricingMode.pricingMode}`,
```
Comment (mohammed-zahaf) [inline packages/manager/modules/netapp/src/order/controller.js:137]: are we sure that `selectedSize` will be all times `TB`

File: packages/manager/modules/netapp/src/dashboard/volumes/dashboard/snapshots/routing.js (hunk 6 above)
Comment (antleblanc) [reply packages/manager/modules/netapp/src/dashboard/volumes/dashboard/snapshots/routing.js:103]: Following call is only used here.

File: packages/manager/modules/netapp/src/dashboard/volumes/dashboard/snapshots/routing.js (hunk 7 above)
Comment (antleblanc) [reply packages/manager/modules/netapp/src/dashboard/volumes/dashboard/snapshots/routing.js:113]: Following call is only used here.

**Comment by antleblanc** [comment]
Translations has been retrieved successfully.

Resolving conflicts for those two files listed below:
- packages/manager/modules/netapp/src/dashboard/snapshot-policies/create/translations/Messages_en_GB.json
- packages/manager/modules/netapp/src/dashboard/volumes/dashboard/translations/Messages_en_GB.json

**Review by JeremyDec — DISMISSED** [review]

**Review by marie-j — DISMISSED** [review]

**Pull request merged**